pkg/webhooks/pod_webhook.go

```diff
@@ -113,28 +120,46 @@ func (p *PodWebhook) Default(ctx context.Context, obj runtime.Object) error {
 		} else {
 			groupUniqueKey = pod.Labels[leaderworkerset.GroupUniqueHashLabelKey]
 		}
-		_, foundEpKey := pod.Annotations[leaderworkerset.ExclusiveKeyAnnotationKey]
-		if foundEpKey && !exclusiveAffinityApplied(*pod) {
-			SetExclusiveAffinities(pod, groupUniqueKey)
+
```

Comment:
line 115 above, can you please collapse it into the if statement: 

```
if _, exists := pod.Labels[leaderworkerset.GroupUniqueHashLabelKey]; !exists { 
```

same thing lines 106 and 107:
```
if _, exists := pod.Labels[leaderworkerset.GroupIndexLabelKey]; !exists {
```

Reply:
I am trying to shorten the larger if/else statement that implements the special behavior for the leader to make it easier to read.

Reply:
done